docs-website/sidebars.js

```diff
@@ -95,6 +95,7 @@ module.exports = {
           "metadata-integration/java/datahub-protobuf/README",
           "metadata-ingestion/as-a-library",
           "metadata-integration/java/as-a-library",
+          //"metadata-ingestion-modules/airflow-plugin/README"
```

Comment:
Do you plan to revert this?

Reply:
I added the doc about this plugin to the original Airflow docs but because I have a README here is as well doc generation complained about having a markdown file that is missing from the sidebar.js. 
As far as I know, this is the way to let doc generation know to ignore a file.